tools/dbTools.py

```diff
@@ -34,7 +34,9 @@ def __del__(self):
         except AttributeError:
             if hasattr(self, 'logger'):
                 # __del__ can be called before self.logger is created.
-                self.logger.warning('Tried to close a non-existent connection.')
+                self.logger.warning(
+                    'Tried to close a non-existent connection.'
+                )
```

Comment:
Ah, yes, our style must be 72 chars. Sorry, will update the vimrc.

Reply:
One last thing we could do is add the command used to push the docker image to AWS into the Makefile.

Reply:
(but only if that's easy)

Reply:
The docker push requires the output from `aws login`, so I don't know how we could do that in an easy, clean way at the moment. Maybe for a separate PR?

Reply:
Methinks everything we need could be set ahead of time in the environment...but I'll chat with you. Good to merge this meantime, one monkey don't stop the show :-)

Reply:
Um, I think I see what you mean by settings things ahead, but still good to have a chat to be sure 👍